<ui version="4.0" >
 <class>PluginsConfigWidget</class>
 <widget class="QWidget" name="PluginsConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>465</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QTableWidget" name="tableWidget" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Plugin</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Enabled</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Author(s):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="authorsLabel" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>E-mail:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="emailLabel" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Homepage:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="homepageLabel" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Copyright:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLabel" name="copyrightLabel" />
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>License:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLabel" name="licenseLabel" />
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="descriptionLabel" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
